### PR TITLE
Span Categorization in prodigy

### PR DESCRIPTION
**Pull request by thomashacker**
This spaCy project showcases how to annotate data in prodigy for Span Categorization. It includes the following features:

### Recipes
- `spans.manual`
- `spans.correct`
- `train`
- `db-out`

### Features
- Instructions
- Patterns
- Suggester Validation

The data is an excerpt of the [Food.com kaggle dataset ](https://www.kaggle.com/datasets/irkaal/foodcom-recipes-and-reviews) and contains 25.000 recipes in `.jsonl` format

### CODE REVIEW

File: tutorials/spancat_food_ingredients/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### 🗂 Assets
+
+The following assets are defined by the project. They can
+be fetched by running [`spacy project assets`](https://spacy.io/api/cli#project-assets)
+in the project directory.
+
+| File | Source | Description |
+| --- | --- | --- |
+| [`assets/food_recipes.jsonl`](assets/food_recipes.jsonl) | Local | Extract of the [Food.com Recipe & Review](https://www.kaggle.com/datasets/irkaal/foodcom-recipes-and-reviews) dataset with 25.000 entries. |
```
Comment (svlandeg) [inline tutorials/spancat_food_ingredients/README.md:41]: Are we allowed to redistribute this?

File: tutorials/spancat_food_ingredients/project.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    script:
+      - "python -m prodigy spans.manual ${vars.database} en_core_web_sm assets/food_recipes.jsonl --label INGREDIENT,INSTRUCTION --suggester custom_suggester -F scripts/suggester.py"
+    deps:
+      - "assets/food_recipes.jsonl"
+      - "scripts/suggester.py"
+
+  - name: "train_spancat"
+    help: "Train a spancat model."
+    script:
+      - "python -m prodigy train ./training --spancat ${vars.database} --eval-split ${vars.eval_split} --gpu-id ${vars.gpu_id}"
```
Comment (svlandeg) [inline tutorials/spancat_food_ingredients/project.yml:62]:
```suggestion
    script:
      - "python -m prodigy train ./training --spancat ${vars.database} --eval-split ${vars.eval_split} --gpu-id ${vars.gpu_id}"
    outputs:
      - "training/model-best"
```

File: tutorials/spancat_food_ingredients/project.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      - "assets/food_recipes.jsonl"
+      - "training/model-best"
+
+  - name: "db_drop"
+    help: "Drop the prodigy database defined in the project.yml"
+    script:
+      - "python -m prodigy drop ${vars.database}"
+
+  - name: "db_export"
+    help: "Export the database defined in the project.yml to .spaCy files"
```
Comment (svlandeg) [inline tutorials/spancat_food_ingredients/project.yml:78]:
```suggestion
    help: "Export the database defined in the project.yml to `.spacy` files"
```

**Review by svlandeg — COMMENTED** [review]
Great supporting material for the upcoming spancat video! 🎉

File: tutorials/spancat_food_ingredients/README.md (hunk 1 above)
Comment (thomashacker) [reply tutorials/spancat_food_ingredients/README.md:41]: The license is `CC0 1.0`
https://www.kaggle.com/datasets/irkaal/foodcom-recipes-and-reviews

**Pull request merged**